libc/getpw.c

```diff
@@ -0,0 +1,111 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libc-tests
+ *
+ * Testing getpwuid and getpwnam edge cases
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Mateusz Niewiadomski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+/* make compilable against glibc (all these tests were run on host against libc/linux kernel) */
```

Comment:
is this true or was it copied from the other C file?

(I believe it's false as the consts You're checking against are not the defaults found on linux)

Reply:
Yes you are right. Leftover copied from other file. I will remove that.